.github/workflows/build-test-verilator.yml

```diff
@@ -13,8 +13,7 @@ env:
   CARGO_INCREMENTAL: 0
   SCCACHE_VERSION: 0.3.3
   RISCV_VERSION: v12.1.0
-  # TODO: To update to 5.006, clean up lint errors
-  VERILATOR_VERSION: v5.002
+  VERILATOR_VERSION: v5.010
```

Comment:
The [README](https://github.com/chipsalliance/caliptra-rtl/blob/d334c84a0e513a3d66a51078f020a414a2c86a67/README.md?plain=1#L34-L35) also needs to be updated.

Reply:
Thanks, updated!